.gitmodules

```diff
@@ -1,23 +1,23 @@
 [submodule "MCServer/Plugins/Core"]
-	path = MCServer/Plugins/Core
+	path = Server/Plugins/Core
 	url = https://github.com/mc-server/Core.git
 [submodule "MCServer/Plugins/ProtectionAreas"]
```

Comment:
What about this
http://stackoverflow.com/questions/4526910/rename-a-git-submodule
it tells us that we have to *sync* the submodules